CMakeLists.txt

```diff
@@ -2719,9 +2698,9 @@ string(REPLACE "\n" "\n " CPACK_DEBIAN_PACKAGE_DESCRIPTION_MERGED "${CPACK_DEBIA
 # The upstream version must not contain hyphen
 string(REPLACE "-" "~" CPACK_DEBIAN_UPSTREAM_VERSION "${MIXXX_VERSION}")
 set(CPACK_DEBIAN_DEBIAN_VERSION 0ubuntu1)
-set(CPACK_DEBIAN_DEBIAN_VERSION_EXTRA "${GIT_BRANCH}~git${GIT_COMMIT_COUNT}")
+set(CPACK_DEBIAN_DEBIAN_VERSION_EXTRA "${GIT_BRANCH}~git${GIT_DESCRIBE}")
```

Comment:
Are there special requirements that might prevent usage of git describe? If so, we should document them as a comment.

Reply:
Yes there are. New versions numbers have to be "greater" and there are no dashes allowed. 
I have original planned to dive into it and come up with a proposal.

Reply:
How about decouple this question from this PR and keep GIT_COMMIT_COUNT unit we have everything sorted out.

Reply:
The version test is: 
```
dpkg --compare-versions 2.0.0~rc1-0ubuntu1 lt 2.0.0-0ubuntu1 && echo true
```

Reply:
> How about decouple this question from this PR and keep GIT_COMMIT_COUNT unit we have everything sorted out.

Or I can propose a PR against this branch. What do you think?

Reply:
I just removed the commit from this PR and we can take care of it in a separate PR.